### PR TITLE
Add radio_locale support

### PR DESCRIPTION
**Pull request by oskarpearson**
Related change: https://github.com/openaps/openaps/pull/112

I don't currently have a box to test this on, but "it should work" (TM). I'll try sort one out this evening.

### CODE REVIEW

File: bin/oref0-setup.sh [hunk 1]
```diff
@@ -377,14 +403,14 @@ if [[ "$ttyport" =~ "spi" ]]; then
...
-if [[ -z "$ttyport" ]]; then
+if [[ -z "$using_mmeowlink" ]]; then
     openaps device add pump medtronic $serial || die "Can't add pump"
     # carelinks can't listen for silence or mmtune, so just do a preflight check instead
     openaps alias add wait-for-silence 'report invoke monitor/temp_basal.json'
     openaps alias add wait-for-long-silence 'report invoke monitor/temp_basal.json'
     openaps alias add mmtune 'report invoke monitor/temp_basal.json'
 else
-    openaps device add pump mmeowlink subg_rfspy $ttyport $serial || die "Can't add pump"
+    openaps device add pump mmeowlink subg_rfspy $ttyport $serial $radio_locale || die "Can't add pump"
```
Comment (scottleibrand) [inline bin/oref0-setup.sh]: We also need to update oref0-setup to install the version of openaps with radio_locale support before this will work, right?

**Comment by oskarpearson** [comment]
I've made some additional changes here - so as to support MDT cgms. I think we can merge.

Currently the end-result doesn't work, since the distributed openaps version doesn't include the requisite changes, but the command run from the scripts is as follows:

`+ openaps device add cgm mmeowlink subg_rfspy /dev/ttyACM0 123123 WW`
and
`+ openaps device add cgm mmeowlink subg_rfspy /dev/ttyACM0 123123`

Currently these extra parameters are  just ignored and not written to the pump.ini / cgm.ini

**Comment by scottleibrand** [comment]
Oh, nice.  I like backwards compatibility.  :)

**Comment by scottleibrand** [comment]
I think what I'll do for now, given the backwards compatibility, is just merge this without any special code to ensure that the user has the openaps dev branch.  Anyone wishing to test this with oref0-setup will just need to check out and install the openaps dev branch manually before running oref0-setup.

**Comment by scottleibrand** [comment]
FWIW, looks like the openaps stuff is actually *not* backwards compatible.  With the dev branch of oref0-setup, you also have to install the dev branch of openaps or it errors out when trying to pass the WW or US flag.

**Pull request merged**